.github/workflows/publish.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+    push:
+        tags:
+            - v*
+
+jobs:
+    build:
+        name: Build & Publish
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+        steps:
+            - name: Checkout
+              uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+            - name: Install Kalamine
+              run: pipx install kalamine
+            - name: Build
+              run: kalamine build finner.toml
+            - name: Release
+              uses: softprops/action-gh-release@c95fe1489396fe8a9eb87c0abf8aa5b2ef267fda # v2.2.1
```

Comment:
Nice convention 👍 I've always liked to do the same thing (pin the immutable hash + comment the tag) for 3rd party packages.